libnavigation-ui/src/main/java/com/mapbox/navigation/ui/VoiceInstructionCache.java

```diff
@@ -81,4 +88,9 @@ void update(int voiceInstructionsToAnnounce) {
       isVoiceInstructionsToCacheThresholdReached = true;
     }
   }
+
+  @TestOnly
+  int getTotalVoiceInstructionNumber() {
```

Comment:
Can't we test using the `public` APIs available? I'm not a big fan of adding explicit "test only" methods. Not a blocker though as this is going to be eventually refactored 👍